tests/rustfmt/main.rs

```diff
@@ -200,3 +200,30 @@ fn rustfmt_emits_error_when_control_brace_style_is_always_next_line() {
     let (_stdout, stderr) = rustfmt(&args);
     assert!(!stderr.contains("error[internal]: left behind trailing whitespace"))
 }
+
```

Comment:
will likely need to rework the tests slightly to accommodate `Error`, `Warn`, and `Ignore` variants.

Reply:
Fixed test on this commit. https://github.com/rust-lang/rustfmt/pull/5611/commits/fcc06f96af07f6423d052191667dc64da2690a25

Reply:
Fixed test on this commit. https://github.com/rust-lang/rustfmt/pull/5611/commits/fcc06f96af07f6423d052191667dc64da2690a25